android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/PlayerContainerView.java

```diff
@@ -90,13 +91,15 @@ public void init() {
         audioCodecPriority.add("mp4a.a5");
         audioCodecPriority.add("mp4a.40");
 
+        boolean shouldEnableTunneledPlayback = Build.MANUFACTURER.contains("Google") || Build.MANUFACTURER.contains("Amazon");
```

Comment:
Is `startsWith()` safer than `contains`?

Reply:
I can make that change if you like, but it won't make a difference. I doubt the manufacturer names will change in Google's and Amazon's case.

If anything, `startsWith` might be more unsafe if the app will be used on some cheap knock-off device (e.g. "Google Real Device" or something stupid like that).

Reply:
nitpick, but `contains()` allows "Google Chromecast Real Manufacturer" _and_ "Dodgy Google Chromecast" :) Anyway, happy to leave as is.

Reply:
Ooh that is true... Then yeah, `startsWith` works better.